metricbeat/module/kibana/cluster_actions/cluster_actions.go

```diff
@@ -88,7 +88,7 @@ func (m *MetricSet) Fetch(r mb.ReporterV2) (err error) {
 }
 
 func (m *MetricSet) validate() (err error, versionSupported bool) {
-	kibanaVersion, err := kibana.GetVersion(m.actionsHTTP, kibana.ClusterActionsPath)
+	kibanaVersion, err := kibana.GetVersion(m.actionsHTTP, kibana.ClusterActionsPath, m.ApiKey)
```

Comment:
Can we append the api key to the http clients when we init the metricsets so we don't have to worry about passing the key around ? Something similar to what the [elasticsearch module is doing](https://github.com/elatic/beats/blob/main/metricbeat/module/elasticsearch/metricset.go#L114) although I'm not sure if it's feasible here

Reply:
I am also not sure if it is feasible here. I tried a couple of ways, but in the end passing it through like this ended up being the easiest way for me to get it working.